cfme/tests/cloud/test_tag_mapping.py

```diff
@@ -262,5 +243,21 @@ def test_ec2_tags_images():
             3. Go to summary of this image  and check whether there is
             test:testing in Labels field
             4. Delete that tag
+        testSteps:
+            1. Create an instance with tag test:testing
```

Comment:
Can you include a comment that you are tagging on the EC2 side and not the CFME side. I initially got confused about why the wrapanapi entities were required for this test case.